tests/integration/create_token.py

```diff
@@ -125,7 +125,11 @@ def _get_payload_with_params(
         payload_vars["response_expires_at"] = get_response_expires_at()
 
         for key, value in extra_payload.items():
-            payload_vars[key] = value
+            if payload_vars.get("survey_metadata"):
+                survey_metadata = payload_vars["survey_metadata"].get("data")
+                survey_metadata[key] = value
```

Comment:
Also, shouldn't this be:
```python3
    data = payload_vars["survey_metadata"].get("data")
    data[key] = value
``` 
for clarity? so:
```python3
data = payload_vars["survey_metadata"].get("data")
       for key, value in extra_payload.items():
                data[key] = value
````

Reply:
Makes sense, updated 👍